content/blog/2024-07-31-tmix-07.md

```diff
@@ -0,0 +1,64 @@
++++
+title = "This Month in Xilem, July 2024"
+authors = ["Daniel McNab", "Olivier Faure"]
++++
+
+This month saw contributions on Linebender projects from volunteer contributors.
+Philipp Mildenberger especially is a major driver for progress on Xilem and especially the `xilem_web` crate.
+
+## Xilem
+
+- [xilem#423][] integrates Tokio into Xilem, showing how the Xilem model can interact with async work.
+- [xilem#428][] by Philipp adds explicit support for Flex parameters.
+- [xilem#440][], [xilem#448], etc add support Async support and experiments in Xilem Web
+
```

Comment:
The calculator example can also go here. That might be worth a screenshot? We want to aim for at least one screenshot in each tmix now, as far as I know.